demos/node-classification/deep-graph-info-max/gcn-deep-graph-info-max-cora-example.ipynb

```diff
@@ -0,0 +1,284 @@
+    "\n",
+    "As with all StellarGraph workflows: first we load the dataset, next we create our data generators, and then we train our model. We then take the embeddings created through unsupervised training and predict the node classes using logistic regression."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from stellargraph.mapper import CorruptedNodeSequence, FullBatchNodeGenerator\n",
+    "from stellargraph import StellarGraph\n",
+    "from stellargraph.layer import GCN, DeepGraphInfoMax\n",
+    "\n",
+    "from stellargraph import datasets\n",
+    "from stellargraph.utils import plot_history\n",
+    "\n",
+    "import random\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from matplotlib import pyplot as plt\n",
+    "from sklearn import preprocessing, feature_extraction, model_selection\n",
+    "from sklearn.linear_model import LogisticRegression\n",
+    "from sklearn.manifold import TSNE\n",
+    "from IPython.display import display, HTML\n",
+    "\n",
+    "from tensorflow.keras.optimizers import Adam\n",
+    "from tensorflow.keras.layers import Input, Dense\n",
+    "from tensorflow.keras.callbacks import EarlyStopping\n",
+    "import tensorflow as tf\n",
+    "from tensorflow.keras import Model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "The Cora dataset consists of 2708 scientific publications classified into one of seven classes. The citation network consists of 5429 links. Each publication in the dataset is described by a 0/1-valued word vector indicating the absence/presence of the corresponding word from the dictionary. The dictionary consists of 1433 unique words."
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "dataset = datasets.Cora()\n",
+    "display(HTML(dataset.description))\n",
+    "G, node_subjects = dataset.load()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    " ## Data Generators\n",
+    " \n",
+    " Now we create the data generators using `CorruptedFullBatchNodeGenerator`. `CorruptedFullBatchNodeGenerator` returns shuffled node features along with the regular node features and we train our model to discriminate between the two. \n",
+    " \n",
+    " Note that:\n",
+    " \n",
+    " - We typically pass all nodes to `generator.flow` because this is an unsupervised task\n",
+    " - We don't pass `targets` to `generator.flow` because these are binary labels (true nodes, false nodes) that are created by `CorruptedFullBatchNodeGenerator`"
```

Comment:
Reminder:

```suggestion
    " - We don't pass `targets` to `generator.flow` because these are binary labels (true nodes, false nodes) that are created by `CorruptedGenerator`"
```

and the same for all other mentions of `CorruptedFullBatchNodeGenerator`.